modules/sessions/tests/SessionsControllerTest.php

```
@@ -10,50 +10,91 @@
-        $routes = array_map(fn (string $action): array => $this->guessRouteForControllerAction(SessionsController::class, $action), $actions);
-
-        // Assert
-        self::assertNotEmpty($actions);
-        self::assertCount(count($actions), $routes);
+        /* Act */
+        sort($actions);
 
-        foreach ($routes as $route) {
-            self::assertContains($route['verb'], ['GET', 'POST']);
-            self::assertMatchesRegularExpression('/^[a-z0-9_\/-]+$/', $route['uri']);
+        $missingExpectedActions = [];
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $missingExpectedActions. Keep variable name length under 20. (undefined)

(LongVariable)